cmd/nerdctl/container/container_exec.go

```diff
@@ -88,13 +86,6 @@ func processExecCommandOptions(cmd *cobra.Command) (types.ContainerExecOptions,
 		}
 	}
 
-	_, isTerminal := term.GetFdInfo(os.Stdin)
-	if !flagD {
-		if flagT && flagI && !isTerminal {
-			return types.ContainerExecOptions{}, errors.New("the input device is not a TTY")
-		}
-	}
-
```

Comment:
Maybe add a comment line to explain why there is no need to check `isTerminal`

Reply:
Done